src/amo/pages/Addon/index.js

```diff
@@ -743,7 +750,10 @@ export class AddonBase extends React.Component {
 }
 
 export function mapStateToProps(state, ownProps) {
-  const { slug } = ownProps.match.params;
+  const slug =
+    typeof ownProps.match.params.slug === 'string'
```

Comment:
given that we retrieve the slug based on the routing parameter, we must use the lowercase version, because the only slug valid that should be stored is lowercase.